libs/x-content/src/main/java/org/elasticsearch/common/xcontent/XContentBuilder.java

```diff
@@ -921,6 +923,42 @@ private XContentBuilder value(ToXContent value, ToXContent.Params params) throws
     // Maps & Iterable
     //////////////////////////////////
 
+    public XContentBuilder stringListField(String name, Collection<String> values) throws IOException {
```

Comment:
Nit: maybe just `array(String name, Collection<String> values)` ? Same remark for the other new methods.

Reply:
Can't do that unfortunately because we have:

```
    public XContentBuilder field(String name, Iterable<?> values) throws IOException {
        return field(name).value(values);
    }
```

which collides with it. Same with the other cases, these super generic `?` or `Object` type methods collide with everything.

Reply:
Oh right, I did not see this one. Let's keep like it is then.